var/spack/repos/builtin/packages/py-mvdtool/package.py

```diff
@@ -15,15 +15,31 @@ class PyMvdtool(PythonPackage):
     git      = "https://github.com/BlueBrain/MVDTool.git"
 
     version('develop', branch='master', submodules=True, clean=False)
+    version('2.3.0', tag='v2.3.0', submodules=True, clean=False)
+    version('2.2.1', tag='v2.2.1', submodules=True, clean=False)
```

Comment:
if we are already deploying 2.3.0, is 2.2.1 required?

Reply:
2.3.0 is a big jump (refactoring, moves...), so it is nice to make avail the latest 2.2.x
We can drop the older ones if you want some cleanup

Reply:
If 2.3.0 is sufficient and if no one really required 2.2., I would remove it.